tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
+    "extends": "@box/frontend/ts/tsconfig",
     "compilerOptions": {
-        "outDir": "./es"
+        "baseUrl": ".",
+        "outDir": "es",
+        "rootDir": "src"
     },
-    "extends": "@box/frontend/ts/tsconfig",
-    "files": ["./.storybook/typings.d.ts"]
+    "exclude": ["src/**/*.js", "node_modules", "es", "dist"],
+    "files": [".storybook/typings.d.ts"],
+    "include": ["src/**/*"]
```

Comment:
Rather than using both `exclude` and `include`, could we do `"include": ["src/**/*.ts", "src/**/*.tsx"]`?